hypertrace-view-generator/hypertrace-view-generator/src/test/resources/StructuredTrace-Hotrod.avro

```diff
@@ -0,0 +1,158 @@
:"String"}],"default":null},{"name":"db_type","type":["null",{"type":"string","avro.java.string":"String"}],"default":null},{"name":"url","type":["null",{"type":"string","avro.java.string":"String"}],"default":null},{"name":"params","type":["null",{"type":"string","avro.java.string":"String"}],"default":null},{"name":"sqlstate","type":["null",{"type":"string","avro.java.string":"String"}],"default":null}]}],"default":null},{"name":"service_name","type":["null",{"type":"string","avro.java.string":"String"}],"default":null},{"name":"rpc","type":["null",{"type":"record","name":"Rpc","namespace":"org.hypertrace.core.datamodel.eventfields.rpc","fields":[{"name":"system","type":["null",{"type":"string","avro.java.string":"String"}],"default":null},{"name":"service","type":["null",{"type":"string","avro.java.string":"String"}],"default":null},{"name":"method","type":["null",{"type":"string","avro.java.string":"String"}],"default":null}]}],"default":null}]}}},{"name":"attributes","type":["null","Attributes"],"default":null},{"name":"start_time_millis","type":"long","default":-1},{"name":"end_time_millis","type":"long","default":-1},{"name":"metrics","type":["null","Metrics"],"default":null},{"name":"entity_edge_list","type":{"type":"array","items":{"type":"record","name":"Edge","fields":[{"name":"edge_type","type":{"type":"enum","name":"EdgeType","symbols":["EVENT_EVENT","ENTITY_ENTITY","ENTITY_EVENT"]}},{"name":"src_index","type":"int","doc":"depending on the EdgeType, this index can point to the element in the entity_nodes or the event_nodes array"},{"name":"tgt_index","type":"int","doc":"depending on the EdgeType, this index can point to the element in the entity_nodes or the event_nodes array"},{"name":"attributes","type":["null","Attributes"],"doc":"Edge attributes that are common between the two nodes. request/response  e.g. api_name, url etc","default":null},{"name":"start_time_millis","type":["null","long"],"default":null},{"name":"end_time_millis","type":["null","long"],"default":null},{"name":"metrics","type":["null","Metrics"],"default":null}]}}},{"name":"event_edge_list","type":{"type":"array","items":"Edge"}},{"name":"entity_event_edge_list","type":{"type":"array","items":"Edge"}},{"name":"entity_entity_graph","type":["null",{"type":"record","name":"Graph","fields":[{"name":"root_nodes","type":{"type":"array","items":"int"},"default":[]},{"name":"adjacency_list","type":{"type":"array","items":{"type":"array","items":"int"}},"default":[]}]}],"default":null},{"name":"event_event_graph","type":["null","Graph"],"default":null},{"name":"entity_event_graph","type":["null","Graph"],"default":null},{"name":"timestamps","type":["null",{"type":"record","name":"Timestamps","fields":[{"name":"records","type":{"type":"map","values":{"type":"record","name":"TimestampRecord","fields":[{"name":"name","type":{"type":"string","avro.java.string":"String"}},{"name":"timestamp","type":"long"}]},"avro.java.string":"String"},"default":{}}]}],"default":null}]} �H��
```

Comment:
Can you add JSON for this too as in one place you have added both avro and json file?